test/models/payment_test.rb

```diff
@@ -1,7 +1,37 @@
 require "test_helper"
 
 class PaymentTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+
+  #-------------------
+  # Refund tests setup
+  #-------------------
+
+  def payment_result(*key_value)
```

Comment:
You need argument for this method only to apply custom id?

If so you should do it like this

```ruby
def payment_result options = {}
id = options.delete(:id) || '385132'

@hash = {"id"=> id, "acquirer"=>"xml-sim".......}

# then you don't need @hash[key_value[0]] = key_value[1]
# also use local variable here -> hash not @hash
end
```